src/esp/assets/PTexMeshData.h

```diff
@@ -22,7 +22,7 @@ namespace assets {
 class PTexMeshData : public BaseMesh {
  public:
   struct MeshData {
-    std::vector<vec4f> vbo;
+    std::vector<vec3f> vbo;
     std::vector<vec4f> nbo;
```

Comment:
Couldn't nbo also be a vec3?

Reply:
As far as I can see, neither `nbo` nor `cbo` is used right now (only populated). Is that correct?

Reply:
> As far as I can see, neither `nbo` nor `cbo` is used right now (only populated). Is that correct?

Yes, you are right. I checked them as well before this PR.